Sonoff iFan support

netRunner0 opened the pull request:
Hi, please add sonoff iFan)

howanghk committed:
support login with phone number / email and password

howanghk committed:
support heartbeat on the WebSocket connection

howanghk committed:
update README and package.json, publish to npm

howanghk committed:
fix heartbeat not send after reconnect

howanghk committed:
fix re-login and re-connect handling

howanghk committed:
0.1.6

howanghk committed:
obtain api and websocket host automatically. Charles is no longer needed!

howanghk committed:
0.1.7

howanghk committed:
fix apiHost undefined bug

howanghk committed:
0.1.8

janbuecker committed:
Add timeout for sending new power states

howanghk committed:
Merge pull request #1 from janbuecker/fix-grouped-sockets

Add timeout for sending new power states

howanghk committed:
0.1.9

aremishevsky commented:
@MrTomAsh could you please review this PR? It is a great feature to prevent hacks with the charles to get token

Jaguaza commented:
Buenos días, este pluging funcionará con un sonoff T1 Eu 2 ganag.

Good morning, this plugin works with a son1 T1 Eu 2 gain.

ramsesz committed:
Catch error when statusupdate is failing

When an error occurs at getting the status like

mebridge/node_modules/homebridge-ewelink-max/index.js:365
    const status = channel && device.params.switches && device.params.switches[channel-1] ? device.params.switches[channel-1].switch : device.params.switch || "off";
                                                                                                                                                     ^
TypeError: Cannot read property 'switch' of undefined

I updated to catch the error otherwise complete homebridge setup fails to run

howanghk committed:
Merge pull request #2 from ramsesz/patch-2

Catch error when statusupdate is failing

howanghk committed:
0.1.10

howanghk committed:
update README, clarify config options
